v2/brigadier/src/workers.ts

```diff
@@ -25,4 +25,33 @@ export interface Worker {
    * configuration.
    */
   logLevel?: string
+  /**
+   * If applicable, contains git-specific Worker details.
+   */
+  git?: GitConfig
+}
+
+/**
+ * Represents git-specific Worker details.
+ */
+export interface GitConfig {
+  /**
+   * Specifies the remote repository where, if applicable, the Worker will have
+   * obtained source code from.
+   */
+  cloneURL: string
+	/**
+   * Specifies the exact commit where, if applicable, the Worker will have
+   * obtained source code from.
+   */
+	commit?: string	
```

Comment:
nit: spacing inconsistency

Reply:
Something must be wrong in my editor...

Reply:
Tabs snuck in somehow even though .ts files are set to use space. 🤷 

Fixing it.